src/enki_files_valiadtor.py

```diff
@@ -189,13 +37,19 @@ def validate(all_files, report, undefined_content, prefix_assemblies, prefix_mod
 
     for path in all_files:
         relative_path = os.path.relpath(path, cwd)
+
         with open(path, 'r') as file:
             original = file.read()
             stripped = Regex.MULTI_LINE_COMMENT.sub('', original)
             stripped = Regex.SINGLE_LINE_COMMENT.sub('', stripped)
-            stripped = Regex.CODE_BLOCK_DASHES.sub('', stripped)
-            stripped = Regex.CODE_BLOCK_TWO_DASHES.sub('', stripped)
-            stripped = Regex.CODE_BLOCK_DOTS.sub('', stripped)
+
+            # this check should run before
+            # code blocks
+            # internal/single line conditionals
+            # are replaced
+            too_many_comments_check(original, stripped, report, path)
```

Comment:
```suggestion
            too_many_comments_check(original, stripped, report, relative_path)
```
This ensures that this test's output is consistent with other tests.